src/Microsoft.AspNetCore.Sockets.Http/HttpConnectionDispatcher.cs

```diff
@@ -385,19 +385,19 @@ private static string GetNegotiatePayload(string connectionId, HttpSocketOptions
                 jsonWriter.WriteStartObject();
                 jsonWriter.WritePropertyName("connectionId");
                 jsonWriter.WriteValue(connectionId);
-                jsonWriter.WritePropertyName("availableTransports");
+                jsonWriter.WritePropertyName("endpoints");
```

Comment:
Endpoints?

Reply:
Yeah, I was playing with naming. I don't really like endpoints either, since it's a concept that exists.

Reply:
metadata?

Reply:
I'm going back to `availableTransports`. It's all still metadata about transports.